web-common/src/features/dashboards/time-dimension-details/context.ts

```diff
@@ -0,0 +1,17 @@
+import type { Writable } from "svelte/store";
+import { getContext, setContext } from "svelte";
+import type { TimeDimensionDetailsStore } from "./time-dimension-details-store";
+import type { THeaderData } from "./mock-data";
+
+const CONTEXT_KEY = "tdt-store";
+
+type TDTContext = {
```

Comment:
Nit: Make the terminology/acronym consistent throughout? `time-details` / TDT (Time DeTails), or `time-dimension-details` / TDD (Time Dimension Details)?

Reply:
Good point, lets go with TDD. I think I started with that

Reply:
FYI I am going to fix this in the next PR